dev-docs/governance.md

```diff
@@ -0,0 +1,117 @@
+# deck.gl Community Governance Guidelines
+
+- Author: Xiaoji Chen
+- Initial draft: December 2019
+- Proposed: March 2020
+- Approved: N/A
+
+
+## Overview
+
+There are three types of roles in the community that maintains this project:
+
+- *Steering committee member*: A core maintainer of the project, provides leadership and oversees feature and technical directions.
+- *Committer*: A maintainer of the project, develops one or more feature areas, and shares the responsibility of release. Also referred to as a "feature owner."
+- *Contributor*: A casual participant on GitHub, including reporting bugs; answering questions; developing a third-party module that extends the capabilities of ours; creating public examples; submitting a PR for documentation enhancement or minor bug fix. Always welcome.
+
+The list of current steering committee members and committers should be documented in the project’s [developer guide](/dev-docs/README.md).
+
+
+## The Steering Committee
+
+### 1a. The steering committee’s responsibilities
+
+- Manage project members and privileges;
+- Ensure that the dev tools of the repo work consistently and as expected;
+- Organize and moderate community discussions;
+- Ensure that the dev process is followed by all collaborators;
+- Ensure the consistency and quality of the documentation;
+- Manage blog posts, material for conference and meetup talks, and other brand presence.
+
+### 1b. Becoming a steering committee member
+
+A steering committee member should be
+
+- A long time contributor to the project;
+- Recognized by the community for their depth of knowledge and community engagement;
+- Agrees to dedicate time to perform their committee duties.
+
+An existing committer can be nominated to join the steering committee by other steering committee members. The nomination is formally submitted as a PR to update the contribution guide. Nominations are approved by the steering committee.
+
+###  1c. Leaving the steering committee
+
+If a steering committee member can/will no longer perform their duties, they can be removed from the steering committee by the consensus of the rest of the steering committee members.
+
+
+## Committer
+
+### 2a. A committer's previledges
+
+- Roadmap planning
+  + Submit, approve and comment on RFCs.
+  + Participate in the planning meeting at the beginning of each release cycle (3a).
+  + Join the developer channel for the ongoing technical discussions, development activities and release updates.
+- Write permission to the repo, and membership to the NPM organization
```

Comment:
Should there be a section on "Steering Committee Members Privileges", in particular regarding Admin permissions to the repos etc? 

I assume all Steering Committee Member should be e.g. github admins - if not them, then who?